### ISSUE TITLE
La aplicación se desvincula aleatoriamente

### ISSUE THREAD

**Issue by m3ck0** [wppconnect-team/wppconnect#1865]
## Description
La sesión se desconecta aleatoriamente cuando reinicio la aplicación.(_The session disconnects randomly when I restart the application._)

Este comportamiento no es esperado porque normalmente en Whatsapp web no tengo que volver a sincronizar cuando reinicio la computadora. _(This behavior is not expected because normally in Whatsapp web I don't have to re-pair when I restart the computer)_

¿Qué está sucediendo en este caso que causa esto y cómo puedo manejarlo, si tiene alguna sugerencia? _(What is happening in this case which causes this and how I can handle it, if you have any suggestions?)_

## Environment
- **WPPConnect version(s):** 1.28.0
- **WA-JS version(s):** 2.24.8
- **Browser:** Chrome XX / Chromium XX (latest, as per this [repo](https://github.com/wppconnect-team/wppconnect-server) tag: `v.2.2.5`)
- **OS:** Ubuntu 22
- **Node version:** Node 18.18 (docker `node:lts-alpine-18`)
- **WhatsApp version:** 2.23.19.84 (Sep28, 2023)
- **MultiDevice (BETA):** no

## Steps to Reproduce
1. empareje la aplicación con wppconnect usando la funcionalidad webhook (_pair app with wppconnect using webhook functionality_)
2. reinicie el contenedor Docker varias veces con varias horas de intervalo (_restart Docker container multiple times with several hours interval_)
3. El WPPConnect se desempareja (_WPPConnect unpairs_)

## Log Output
Los estados de la sesión tienen este orden después del reinicio. (_Session states have this order after reboot._)
1. `inChat`
2. `desconnectedMobile`
3. `browserClose`
4. `notLogged`
5. `qrReadError`
6. `browserClose`

## Your Code
No tengo ninguna modificación de código excepto la configuración.(_I don't have any code modification except configuration._)

Aquí solo se enumeran los valores modificados en el archivo [config.ts](https://github.com/wppconnect-team/wppconnect-server/blob/main/src/config.ts) (_Only the modified values in the file are listed here_):
```
{
    ...
    deviceName: 'Desktop',
    webhook: {
       url: 'link to my webhook',
    }
}
```

## Additional context / Screenshot
- Estoy usando Docker con la funcionalidad de webhook habilitada (_I am using Docker with webhook functionality enabled_)
- La persistencia del volumen está configurada correctamente, porque en la mayoría de los reinicios la sesión se vuelve a conectar. (_Volume persistence is configured correctly, because on most reboots the session reconnects._)

**Comment by momokang** [comment]
Yes, I do have this issue and it occur very randomly, last time is RAM issue so I upgraded my RAM, but now RAM is stable and it is still happening, do anyone have idea how to solve it?
I would like it to connect without logout randomly.

**Comment by icleitoncosta** [comment]
> Yes, I do have this issue and it occur very randomly, last time is RAM issue so I upgraded my RAM, but now RAM is stable and it is still happening, do anyone have idea how to solve it?
> I would like it to connect without logout randomly.

Could you post your machine's configurations here?

**Comment by m3ck0** [comment]
@icleitoncosta what kind of configuration details you'd like to know? For resources we have tried really high-end machines (4vCPU, 16GB memory) for a single device to make sure that the resources were not the problem & utilization was always low.
